### DISCUSSION TITLE
GetNextPositionOnStreet on the nightly build not behaving correctly

### DISCUSSION THREAD

**Discussion by DealTheAl** [scripthookvdotnet/scripthookvdotnet#1461] [Q&A] [answered]
When using the function:      **World.GetNextPositionOnStreet(Vector3 position, [bool unoccupied = false])** 
It always  **return a vector 0 if unoccupied is "false"**.  Everything seems working fine if unoccupied is set to "true".
I still can't understand why. Hopefully it can be fixed.

Thanks in advance.

**Comment by kagikn** [comment, marked as answer]
Created #1463 as a bug report and now closed with a fix. Try [nightly.106](https://github.com/scripthookvdotnet/scripthookvdotnet-nightly/releases/tag/v3.6.0-nightly.106) or later.
